fsharp-backend/tests/testfiles/string.tests

```diff
@@ -52,6 +52,7 @@ String.replaceAll_v0 "abcABCcbaCBA" "b" "x" = "axcABCcxaCBA"
 // String.slice_v0 "abcd" 2 6 = "cd"
 // String.slice_v0 "abcd" 3 2 = ""
 // String.slice_v0 "abcd" 5 6 = ""
+String.slugify_v0 "  m@y  'super'  really- excellent *uber_ amazing* ~very  ~ \"clever\" thing: coffee😭! " = "my-super-really-excellent-uber-amazing-very-clever-thing-coffee" // Slugify works
```

Comment:
This works too well for slugify_v0 I think. the goal is to have it continue to work exactly the same as in OCaml. Can you check the output of the ocaml version on darklang.com?

Reply:
This is the result, 
![image](https://user-images.githubusercontent.com/13671857/102591256-97d5b700-40c6-11eb-9e0c-51ad8c39676a.png)